WTWSMS/localisation/WtWSMS_religions.csv

```diff
@@ -143,10 +143,14 @@ temple_empire_of_sethian;Snake See of;
 d_sethian_holy_order;The Knights Ouroboros;Les chevaliers Ouroboros ;;;;;;;;;x
 d_sethian_holy_order_adj;Snake Knight;Chevalier serpent;;;;;;;;;x
 ###### nicene;;;;;;;;;;;;x
-nicene;Nicene;nic�en;;;;;;;;;;x
-nicene_DESC;Born out of the Council of Nicaea in 325, the Nicene Christianity is the central church of the Pentarchy, and the main faith of the two empires.;N�e lors du concile de Nic�e en 325, le christianisme nic�en est l'�glise centrale de la Pentarchie, et la foi principale des deux empires.;;;;;;;;;;x
+#nicene;Nicene;nic�en;;;;;;;;;;x
```

Comment:
Damn, apparently broke encoding. Will fix.

Reply:
I still don't see why some "é"s appear as "é" and some as "�", even in the unchanged lines (for example: d_orthodox;Ecumenical Patriarchy;Patriarcat �cum�nique;;;;;;;;;;x
d_orthodox_adj;Ecumenical;Œcuménique;;;;;;;;;;x)
@loup99 any idea

Reply:
No idea.

Reply:
I tried redoing the changes in another branch but the result was the same.
I run the game with French language, however, and the names are fine in the game:
![image](https://user-images.githubusercontent.com/29546927/71815175-e7220380-307e-11ea-95a9-c2a5fc670bf1.png)

Reply:
Yes, that works exactly as it should (except for the second grammar mistake I spotted 😛 ).